ten-simple-rules-dockerfiles.tex

```diff
@@ -331,10 +331,9 @@ \section*{Introduction}\label{introduction}}
 {[}3,4{]}; an overall goal of these practices is to ensure that both the
 author and others are able to reproduce and extend an analysis workflow.
 The containers built from these recipes are portable encapsulated
-snapshots of a specific computing environment, which are both more
-lightweight and transparent than virtual machines. Such containers have
-been demonstrated for capturing scientific notebooks {[}5{]} and
-reproducible workflows {[}6{]}.
+snapshots of a specific computing environment. Such containers have been
+demonstrated for capturing scientific notebooks {[}5{]} and reproducible
+workflows {[}6{]}.
```

Comment:
I would still leave this clause in since I think many readers will understand the meaning and it is good to promote the idea of containers.

Reply:
Also, this change only appears to be in the `.tex` file, not the `.Rmd` file.